js/private/js_binary.sh.tpl

```diff
@@ -118,6 +118,11 @@ trap _exit EXIT
 # ==============================================================================
 {{initialize_runfiles}}
 
+# TODO(2.0): export only JS_BINARY__RUNFILES
+export RUNFILES
+JS_BINARY__RUNFILES="$RUNFILES"
+export JS_BINARY__RUNFILES
```

Comment:
hahaha everywhere that already had to special-case JAVA_RUNFILES https://bazel.build/reference/test-encyclopedia